Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -1058,7 +1058,7 @@ void EmuCodeBlock::SetFPRF(Gen::X64Reg xmm)
 void EmuCodeBlock::JitGetAndClearCAOV(bool oe)
 {
 	if (oe)
-		AND(8, PPCSTATE(xer_so_ov), Imm32(~XER_OV_MASK)); //XER.OV = 0
+		AND(8, PPCSTATE(xer_so_ov), Imm8(~XER_OV_MASK)); //XER.OV = 0
```

Comment:
Yeah, certain no guarantee our overflow handling is 100% correct even for the parts we do implement, but that was just a dumb typo in an older patch of mine months ago which never triggered because nothing used that code.